doc/Example of Cross Compiling the Azure IoT SDK for Python.md

```diff
@@ -0,0 +1,230 @@
+# Cross Compiling Azure IoT SDK for Python Example
+## Overview
+
+The goal of this document is to describe the steps required to cross compile the Azure IoT SDK for Python in order to allow the user to use Python to send and receive messages between the device and one&#39;s Azure IoT Hub. For this example, the target device will be the [MultiTech device](https://catalog.azureiotsuite.com/details?title=MultiConnect-Conduit&amp;source=home-page). It will describe how to install the toolchain for the this device, build the required prerequisite software and finally cross compile the Azure IoT SDK for Python library. The procedure for other Linux based target devices should be very similar.
```

Comment:
>toolchain [](start = 419, length = 9)

probably better: "compiler toolchain"

Reply:
Agreed

---
In reply to: [142532030](https://github.com/Azure/azure-iot-sdk-python/pull/76#discussion_r142532030) [](ancestors = 142532030)